microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
+Regardless of the rating given to your extension, it must still adhere to all the content requirements of the Microsoft Edge Add-ons store developer policies.
+
 
-#### 2.12.2 Ratings Change
+###### 2.12.2 Ratings change
 
 If your extension provides content (such as user-generated, retail, or other web-based content) that might be appropriate for a higher age rating than the assigned rating, you must require users to opt into receiving such content by using a content filter or by signing in with a pre-existing account.
 
-### 2.13 Videos
 
-If you submit a promotional video in the listing, it should follow all the content guidelines mentioned in this policy.  If you choose to provide a YouTube link, you must ensure that advertisements are turned off for the specific videos that you want to embed.  For more information on turning on or off ads on YouTube, see [Set your default ad formats](https://support.google.com/youtube/answer/2531367?ref_topic=7072227) and [Ads on embedded videos](https://support.google.com/youtube/answer/132596).
+<!-- ------------------------------ -->
+#### 2.13 Videos
 
+If you submit a promotional video in the listing, it should follow all the content guidelines mentioned in this policy.  If you choose to provide a YouTube link, you must ensure that advertisements are turned off for the specific videos that you want to embed.  For more information on turning on or off ads on YouTube, see [YouTube advertising formats](https://support.google.com/youtube/answer/2531367?ref_topic=7072227) and [Ads on embedded videos](https://support.google.com/youtube/answer/132596).
 
-<!-- ====================================================================== -->
-## Edge Add-ons Certification Complaint and Appeal Process
 
-All the extensions should adhere to the store policies listed above. If your extension failed in the review process, please review the store policies to understand the reason for failure. After submitting your extension using Partner Center, to ask a question about the review or certification status of it, go to [New Support Request](https://support.microsoft.com/supportrequestform/e7a381be-9c9a-fafb-ed76-262bc93fd9e4) and complete the form.
+<!-- ====================================================================== -->
+## Complaint and appeal process for Microsoft Edge Add-ons certification 
 
-### Microsoft Edge Add-ons appeal statistics for FY2022
+All extensions must adhere to the store policies listed above.  If your extension failed in the review process, review the store policies to understand the reason for failure.  After submitting your extension using Partner Center, to ask a question about the review or certification status of it, go to [New Support Request](https://support.microsoft.com/supportrequestform/e7a381be-9c9a-fafb-ed76-262bc93fd9e4) and complete the form.
```

Comment:
"Partner Center" occurs 5 times in this article.
4 times are linked - that's excessive linking; linking every mention for no reason.  The present instance is not linked, and that works well.  Recommend un-linking some of the 4 instances.  

And they are not a short Partner Center URL, they are a link that took a long time to resolve for me, and then rejected my account:
https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd

Reply:
keeping this instance non-linked